edweiss-app/app/(app)/quiz/createQuizPage.tsx

```diff
@@ -615,8 +738,19 @@ const PropositionField: ReactComponent<{ proposition: [string, boolean], changeP
 export const TFFields: ReactComponent<{ editExercise?: (exercise: Quizzes.Exercise, index: number) => void, addToExerciseList?: (exercise: Quizzes.Exercise) => void, modalRef?: RefObject<BottomSheetModalMethods>, previousQuestion?: string, previousBoolean?: boolean, index?: number }> = (props) => {
 	const [question, setQuestion] = useState<string>(props.previousQuestion == undefined ? "" : props.previousQuestion);
 	const [answer, setAnswer] = useState<boolean>(props.previousBoolean == undefined ? true : props.previousBoolean);
+	const [questionError, setQuestionError] = useState<string | undefined>(undefined);
 	const placeHolder = props.previousQuestion == undefined ? 'Is the earth flat?' : props.previousQuestion
 
+	function handleQuestionInputError(text: string) {
+		if (text.length == 0) {
+			setQuestionError("Please write a question")
+		}
+		else {
+			setQuestionError(undefined)
+		}
+
+	}
```

Comment:
how do you use this function ? If you want to make it change dynamically maybe use the hook `useEffect` so that the questionError changes automatically when the text length is 0 or not. If this is not the behaviour you want, ignore my comment

Reply:
You are right, I will use useEffect.

Reply:
ok I see, so leave it like this then